src/main/java/zowe/client/sdk/zosfiles/uss/methods/UssGetAcl.java

```diff
@@ -9,5 +9,94 @@
+import java.util.Map;
+
+/**
+ * Provides Unix System Services (USS) getfacl functionality
+ *
+ * @author James Kostrewski
+ * @version 2.0
+ */
 public class UssGetAcl {
+
+        private final ZosConnection connection;
+        private ZosmfRequest request;
+
+        /**
+        * UssGetAcl Constructor
+        *
+        * @param connection connection information, see ZosConnection object
+        */
+        public UssGetAcl(ZosConnection connection) {
+            ValidateUtils.checkConnection(connection);
+            this.connection = connection;
+        }
+
+        /**
+        * Alternative UssGetAcl constructor with ZoweRequest object. This is mainly used for internal code unit testing
+        * with mockito, and it is not recommended to be used by the larger community.
+        *
+        * @param connection connection information, see ZosConnection object
+        * @param request any compatible ZoweRequest Interface object
+        * @throws Exception processing error
+        */
+        public UssGetAcl(ZosConnection connection, ZosmfRequest request) throws Exception {
+            ValidateUtils.checkConnection(connection);
+            ValidateUtils.checkNullParameter(request == null, "request is null");
+            this.connection = connection;
+            if (!(request instanceof PutJsonZosmfRequest)) {
+                throw new Exception("PUT_JSON request type required");
+            }
+            this.request = request;
+        }
+
+        /**
+        * Get the ACL for a USS file or directory
+        *
+        * @param fileNamePath file name with path
+         * @param params GetAclParams object to drive the request
+        * @return Response object
+        * @throws Exception processing error
+        */
+        public Response getAcl(String fileNamePath, GetAclParams params) throws Exception {
+            ValidateUtils.checkNullParameter(fileNamePath == null, "path is null");
+            ValidateUtils.checkIllegalParameter(fileNamePath.trim().isBlank(), "path is empty");
+            ValidateUtils.checkNullParameter(params == null, "params is null");
+
+            final String url = "https://" + connection.getHost() + ":" + connection.getZosmfPort() +
+                    ZosFilesConstants.RESOURCE + ZosFilesConstants.RES_USS_FILES +
+                    EncodeUtils.encodeURIComponent(FileUtils.validatePath(fileNamePath));
+
+            final Map<String, Object> getFaclMap = new HashMap<>();
+            params.getType().ifPresent(type -> getFaclMap.put("type", type));
+            params.getUser().ifPresent(user -> getFaclMap.put("user", user));
+            if (params.getUseCommas())
+                getFaclMap.put("useCommas", params.getUseCommas());
+            if (params.getSuppressHeader())
+                getFaclMap.put("suppressHeader", params.getSuppressHeader());
+            if (params.getSuppressBaseAcl())
+                getFaclMap.put("suppressBaseAcl", params.getSuppressBaseAcl());
```

Comment:
incorrect key value, it should be "suppress-baseacl" see IBM web documentation please